spacy/lang/nl/__init__.py

```diff
@@ -46,4 +29,24 @@ class Dutch(Language):
     Defaults = DutchDefaults
 
 
+@Dutch.factory(
+    "lemmatizer",
+    assigns=["token.lemma"],
+    default_config={"model": None, "mode": "rule", "lookups": None},
+    scores=["lemma_acc"],
+    default_score_weights={"lemma_acc": 1.0},
+)
+def make_lemmatizer(
+    nlp: Language,
+    model: Optional[Model],
+    name: str,
+    mode: str,
+    lookups: Optional[Lookups],
+):
+    config = dict(LOOKUPS_TABLES_CONFIG)
+    config["rules"] = ["lemma_lookup", "lemma_rules", "lemma_exc", "lemma_index"]
```

Comment:
I think this needs a `"required_tables"` key?

Also I wonder if we could streamline this part a bit so we don't have to keep passing around the `LOOKUPS_TABLES_CONFIG` and copy it, read from nested dicts etc.

There are basically 4 lists here, right? Optional and required tables, for rules and lookups? Maybe we could just make them arguments on `load_lemmatizer_lookups` then?

Reply:
Ah, that didn't get updated correctly, thanks.

Yes, I think it would cleaner to have the required and optional tables as the arguments to `load_lemmatizer_lookups` instead of the whole config.

The reason for passing it around the config (I agree it's clunky) is that you need to know what the tables are when you try to create something like a lookup lemmatizer for `Dutch` without having to add anything to `DutchLemmatizer`. Maybe the defaults should be stored as class variables in `Lemmatizer` instead? I was trying to make it so that you didn't have to override `__init__` or `__call__` when extending the `Lemmatizer`, so for your Dutch rule-based lemmatizer you just had to implement `rule_lemmatize`.

Let me see what I can do...